Data Availability, Funding acknowledgements and CRedIT keywords (#22)
greenelab/annorxiver_manuscript: content/07.acknowledgements.md

> ## Data Availability

Reviewer: ```suggestion
## Software and Data Availability
```

> An online version of this manuscript is available at [https://greenelab.github.io/annorxiver_manuscript/](https://greenelab.github.io/annorxiver_manuscript/). 

Reviewer: ```suggestion
An online version of this manuscript is available under a Creative Commons Attribution License at [https://greenelab.github.io/annorxiver_manuscript/](https://greenelab.github.io/annorxiver_manuscript/). 
```

> Code for this research can be found at [https://github.com/greenelab/annorxiver](https://github.com/greenelab/annorxiver).

Reviewer: ```suggestion
Source for the research portions of this project is dual licensed under the BSD 3-Clause and Creative Commons Public Domain Dedication Licenses at [https://github.com/greenelab/annorxiver](https://github.com/greenelab/annorxiver).
```

> The journal recommendation website can be found at [https://greenelab.github.io/annorxiver-journal-recommender/](https://greenelab.github.io/annorxiver-journal-recommender/) and code for the website can be found at [https://github.com/greenelab/annorxiver-journal-recommender](https://github.com/greenelab/annorxiver-journal-recommender).

Reviewer: ```suggestion
The journal recommendation website can be found at [https://greenelab.github.io/annorxiver-journal-recommender/](https://greenelab.github.io/annorxiver-journal-recommender/) and code for the website is available under a _INSERT LICENSE_ at [https://github.com/greenelab/annorxiver-journal-recommender](https://github.com/greenelab/annorxiver-journal-recommender).
```

> 

Reviewer: Other two elements to add are the data sources for th ework (bioRxiv / PMC OA)

Source for the research portions of this project is dual licensed under the BSD 3-Clause and Creative Commons Public Domain Dedication Licenses at [https://github.com/greenelab/annorxiver](https://github.com/greenelab/annorxiver).
The journal recommendation website can be found at [https://greenelab.github.io/annorxiver-journal-recommender/](https://greenelab.github.io/annorxiver-journal-recommender/) and code for the website is available under a BSD-2-Clause Plus Patent License at [https://github.com/greenelab/annorxiver-journal-recommender](https://github.com/greenelab/annorxiver-journal-recommender).
Full text access for the bioRxiv repository is available at [https://www.biorxiv.org/tdm](https://www.biorxiv.org/tdm).
Access to PubMed Central's Open Access subset is available on NCBI's FTP server at [https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/](https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/).
Access to the New York Times Annotated Corpus (NYTAC) is available upon request with the Linguistic Data Consortium at [https://catalog.ldc.upenn.edu/LDC2008T19](https://catalog.ldc.upenn.edu/LDC2008T19).

## Acknowledgements

> The authors would like to thank Ariel Hippen Anderson for her aide in curating potential missing preprint to published version links.

Reviewer: ```suggestion
The authors would like to thank Ariel Hippen Anderson for evaluating potential missing preprint to published version links.
```

> We also would like to thank Richard Sever and his group for their assistance on download and processing preprints from the bioRxiv repository. 

Reviewer: ```suggestion
We also would like to thank Richard Sever and the _bioRxiv_ team for their assistance with access to and support with questions about preprint full text downloaded from _bioRxiv_. 
```
